locations/spiders/poundland.py

```diff
@@ -0,0 +1,61 @@
+import scrapy
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class PoundlandSpider(scrapy.Spider):
+    name = "poundland"
+    poundland = {"brand": "Poundland", "brand_wikidata": "Q1434528"}
```

Comment:
I don't know exactly what you mean because the only think bing `.copy()`ed is a `GeojsonPointItem`, not brand stuff.

I can drop the `pep` and `poundland` fields though and just do it in `parse` if you'd prefer.

Reply:
> I can drop the pep and poundland fields though and just do it in parse if you'd prefer.

Yes please, that would help with readability.